### PR TITLE
Added Kurdish (Xwarîn) (sdh)

### LINKED ISSUE #2530: Kurdish (Xwarîn) (sdh)
* Language name: Kurdish (Xwarîn) 
* Language info: https://en.wikipedia.org/wiki/Southern_Kurdish
* ISO 639-3 code: SDH
* Public List: https://tatoeba.org/eng/sentences_lists/show/168674
* Icon: https://imgur.com/a/Z2v0vW4
* Requested by: https://tatoeba.org/eng/user/profile/Sarchia

We're checking which sentences on our corpus are in this language.

### PR DESCRIPTION
**Pull request by RyckRichards**
This PR closes #2530.
It adds Kurdish (Xwarîn), one of the languages that will replace Kurdish (kur).

@trang @jiru - should I create a separated PR which deletes Kurdish (kur)?

### CODE REVIEW

File: src/Lib/LanguagesLib.php [hunk 1]
```diff
@@ -612,6 +612,7 @@ public static function languagesInTatoeba()
                 'mni' => __d('languages', 'Meitei'),
                 'ayl' => __d('languages', 'Libyan Arabic'),
                 'ood' => __d('languages', "O'odham"),
+                'sdh' => __d('languages', 'Kurdish (Xwarîn)'
```
Comment (jiru) [inline src/Lib/LanguagesLib.php:615]: You forgot a closing parenthesis.

**Comment by jiru** [comment]
> should I create a separated PR which deletes Kurdish (kur)?

No, it’s okay. :slightly_smiling_face:

File: src/Lib/LanguagesLib.php (hunk 1 above)
Comment (RyckRichards) [reply src/Lib/LanguagesLib.php]: Oh sorry. Thank you!

**Comment by trang** [comment]
@RyckRichards Please change the name to `Southern Kurdish (Xwarîn)` as suggested in https://github.com/Tatoeba/tatoeba2/issues/2529#issuecomment-699597267.

**Pull request merged**